modules/psap-driver-toolkit-using.adoc

```diff
@@ -0,0 +1,208 @@
+
+[source,terminal]
+----
+$ OCP_VERSION=$(oc get clusterversion/version -ojsonpath={.status.desired.version})
+$ DRIVER_TOOLKIT_IMAGE=$(oc adm release info $OCP_VERSION --image-for=driver-toolkit)
+$ sed “s/DRIVER_TOOLKIT_IMAGE/$DRIVER_TOOLKIT_IMAGE/” 0000-buildconfig.yaml.template > 0000-buildconfig.yaml
+----
+[NOTE]
+====
+Note that the driver toolkit was introduced to {product-title} 4.6 as of version 4.6.30, in 4.7 as of version 4.7.11, and in 4.8.
+====
+
+
+Create the ImageStream and BuildConfig with 
+[source,terminal]
+----
+$ oc create -f 0000-buildconfig.yaml
+----
+
+Once the builder pod completes successfully, we can deploy the driver container image as a `DaemonSet`. The driver container needs to run with the privileged security context in order to load the kernel modules on the host. The following .yaml file contains the RBAC rules and the `DaemonSet` for running the driver container. Save this YAML as `1000-driver-container.yaml`.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: simple-kmod-driver-container
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: simple-kmod-driver-container
+rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+  resourceNames:
+  - privileged
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: simple-kmod-driver-container
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: simple-kmod-driver-container
+subjects:
+- kind: ServiceAccount
+  name: simple-kmod-driver-container
+userNames:
+- system:serviceaccount:simple-kmod-demo:simple-kmod-driver-container
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: simple-kmod-driver-container
+spec:
+  selector:
+    matchLabels:
+      app: simple-kmod-driver-container
+  template:
+    metadata:
+      labels:
+        app: simple-kmod-driver-container
+    spec:
+      serviceAccount: simple-kmod-driver-container
+      serviceAccountName: simple-kmod-driver-container
+      containers:
+      - image: image-registry.openshift-image-registry.svc:5000/simple-kmod-demo/simple-kmod-driver-container:demo
+        name: simple-kmod-driver-container
+        imagePullPolicy: Always
+        command: ["/sbin/init"]
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@simple-kmod"]
+        securityContext:
+          privileged: true
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+
+Create the RBAC rules and `DaemonSet` with
+[source,terminal]
+----
+$ oc create -f 1000-drivercontainer.yaml
+----
+
+Once the pods are running on the worker nodes, we can verify that the `simple_kmod` kernel module is loaded successfully on the host machines with `lsmod`.
+[source,terminal]
```

Comment:
Break these up into separate commands and output blocks per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output